src/rgw/store/sfs/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+#need to update cmake version here
+cmake_minimum_required(VERSION 3.14.0)
+project(sfsstore)
+
+option(USE_SFS "Enable SFS" ON)
+
+find_package(SQLite3 REQUIRED)
+set(SQLITE_COMPILE_FLAGS "-DSQLITE_THREADSAFE=1")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SQLITE_COMPILE_FLAGS}")
+
+set (CMAKE_INCLUDE_DIR ${CMAKE_INCLUDE_DIR} "${CMAKE_CURRENT_SOURCE_DIR}/sqlite")
+
+set(sfsstore_sqlite_srcs
+    sqlite/sqlite_users.cc
+    sqlite/sqlite_buckets.cc
+    sqlite/sqlite_objects.cc
+    sqlite/sqlite_versioned_objects.cc
+    sqlite/users/users_conversions.cc
+    sqlite/buckets/bucket_conversions.cc
+    sqlite/objects/object_conversions.cc
+    sqlite/versioned_object/versioned_object_conversions.cc
+    )
+
+set(sfsstore_srcs
+    bucket.cc
+    multipart.cc
+    object.cc
+    user.cc
+    types.cc
+    zone.cc
+    writer.cc
+    sfs_bucket.cc
+    sfs_user.cc
+    )
+
+add_library(sfsstore_sqlite ${sfsstore_sqlite_srcs})
```

Comment:
I just don't understand why we're making our sqlite bits into a library, instead of statically compiling them alongside the rest of the sfs store?

Reply:
This is simply a choice to make things organized this way.

Reply:
I understand that, but on the other hand, given we're not making a library that could potentially be used by other parties, it makes sense to eventually statically compile those sources with the rest of the sfs store bits instead of creating a library. If I understand correctly, that would be just like adding `sfsstore_sqlite_srcs` to the line below at `add_library(sfsstore STATIC ${sfsstore_srcs})`.

Also, something I noticed, is that `sfsstore` is redundant. The last `s` on `sfs` means `store`. So these variables should read `sfstore`, or `sfs`, rather than `sfsstore`.

Reply:
ok, I will try to implement these changes

Reply:
please, check this now.